doc/Learn/Navigation/HowTo-NavigateInXAML.md

```diff
@@ -58,114 +93,92 @@ Depending on the type of the XAML element, the `Navigation.Request` property wil
     > [!TIP]
     > While this works, it relies on reflection to convert the request path "Sample" to the corresponding view, i.e. `SamplePage`. It's better to define `ViewMap` and `RouteMap`
 
-- Add a `ViewMap` and a `RouteMap` for the `SamplePage` into the `RegisterRoutes` method in the `App.xaml.cs` file
+- Add a `ViewMap` and a `RouteMap` for the `SamplePage` into the `RegisterRoutes` method in the `App.xaml.cs` file:
 
-    ```csharp
-    private static void RegisterRoutes(IViewRegistry views, IRouteRegistry routes)
-    {
-        views.Register(
-            new ViewMap(ViewModel: typeof(ShellViewModel)),
-            new ViewMap<MainPage, MainViewModel>(),
-            new DataViewMap<SecondPage, SecondViewModel, Entity>(),
-            new ViewMap<SamplePage, SampleViewModel>()
-        );
-
-        routes.Register(
-            new RouteMap("", View: views.FindByViewModel<ShellViewModel>(),
-                Nested:
-                [
-                    new ("Main", View: views.FindByViewModel<MainViewModel>()),
-                    new ("Second", View: views.FindByViewModel<SecondViewModel>()),
-                    new ("Sample", View: views.FindByViewModel<SampleViewModel>()),
-                ]
-            )
-        );
-    }
-    ```
+#### [Register Routes MVVM](#tab/mvvm/RegisterRoutes)
+
+[!INCLUDE [register-routes-mvvm](./includes/register-routes-mvvm.md)]
+
+#### [Register Routes MVUX](#tab/mvux/RegisterRoutes)
+
+[!INCLUDE [register-routes-mvux](./includes/register-routes-mvux.md)]
+
+---
 
 ### 2. Navigation.Data
 
+#### [MVVM](#tab/mvvm/NavigationData)
```

Comment:
I'm not sure I'm seeing why we need to separate between MVUX and MVVM Toolkit here. The syntax should be almost identical, shouldn't it?

Just need to define a view model that has a constructor parameter for the data